UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1914,6 +1918,8 @@ private void MenuItem_Add_Click(object sender, RoutedEventArgs e)
                     (obj as UndertaleNamedResource).Name = new UndertaleString(notDataNewName); // not Data.MakeString!
                 }
             }
+            else if (obj is UndertaleString str)
+                str.Content = "";
```

Comment:
Would prefer if this would be set to something like "new string" instead of being a literal empty string.

Reply:
If it will do that, then you'll have to erase "new string" each time when you add the string.

Reply:
Which isn't a huge deal. Other stuff, like adding new code/script entries, behave similar.

Reply:
Well, currently when I add a new string, it is added as an empty string, which is suboptimal.
![image](https://user-images.githubusercontent.com/38186597/173902298-434802ae-8cb7-4850-ae1b-901849b19ee9.png)

It would be better if it would start with something else. So i'm pretty sure I am talking about the content here, not name.

Reply:
But how is it better?
When you are creating the string, you should already have a concept what to write to it.
Having "new string" there will lead only to unnecessary actions - selecting all and pressing Delete.

Reply:
And now it's not a problem to have an empty string for some time, as you can properly click to it.

Reply:
You didn't get what I was trying to say.
Yes, having a literal empty string be represented by something else is great and should stay

What I'm asking for, is that when a new name gets *added*, that an actual placeholder string gets created. Currently the string does get created as an empty string, and only gets *displayed* differently, because of the precaution from above. But instead a string similar to "newstring1" or something should get created, similar to what happens if you create a new code/script/object/room. You don't just get a new code entry with an empty name, the program goes out of its way to generate a placeholder like "code1234" so its differentiable. And something like that should happen here as well.